web/src/styles/global.css

```
@@ -193,3 +193,36 @@ html {
+    background: #fff !important;
+    color: #000 !important;
+  }
+
+  .react-flow__controls,
+  .react-flow__minimap,
+  .react-flow__attribution {
+    display: none !important;
+  }
+
+  [data-testid="org-chart-canvas"] {
+    page-break-inside: avoid;
```

Stylelint (17.9.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)